src/wallet/rpcwallet.cpp

```diff
@@ -53,6 +55,7 @@ using namespace libzcash;
 
 const std::string ADDR_TYPE_SPROUT = "sprout";
 const std::string ADDR_TYPE_SAPLING = "sapling";
+const bool DEFAULT_WALLET_REQUIRE_BACKUP = true;
```

Comment:
This is unused, and we don't want to add a direct dependency on the wallet to the chainparams logic, so remove this:
```suggestion
```